### ISSUE TITLE
PhpStorm 2021.1 support

### ISSUE THREAD

**Issue by olivernybroe** [pestphp/pest-intellij#92]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Plugin version   | 0.4.2-alpha.2
| Pest version     | x.y.z
| OS               | Mac OS X

### Description
I m getting the following error when using the pluing with PhpStorm 2021.1

### Stacktrace
```
com.intellij.diagnostic.PluginException: TypeProvider contract violation? by class com.pestphp.pest.types.ThisTypeProvider on test('it sends a notification on the first ranking update', function () {
    $user = User::factory()->create(['username' => 'johndoe', 'email_verified_at' => now()]);

    Notification::fake();
    Artisan::call('ranking:update');
    Notification::assertSentTo($user, FirstMilitaryRanking::class);
}) [Plugin: com.pestphp.pest-intellij]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:300)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:81)
	at com.jetbrains.php.lang.psi.resolve.types.PhpTypeInfo.lambda$getTypeFromAST$1(PhpTypeInfo.java:93)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:42)
	at com.jetbrains.php.lang.psi.resolve.types.PhpTypeInfo.getTypeFromAST(PhpTypeInfo.java:74)
	at com.jetbrains.php.lang.psi.resolve.types.PhpTypeInfo.getType(PhpTypeInfo.java:57)
	at com.jetbrains.php.lang.psi.resolve.types.PhpTypeInfo.getType(PhpTypeInfo.java:31)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.getType(PhpReferenceImpl.java:238)
	at com.jetbrains.php.lang.psi.elements.PhpTypedElement.getGlobalType(PhpTypedElement.java:53)
	at com.jetbrains.php.lang.inspections.PhpVoidFunctionResultUsedInspection.analyzeCall(PhpVoidFunctionResultUsedInspection.java:50)
	at com.jetbrains.php.lang.inspections.PhpVoidFunctionResultUsedInspection$1.visitPhpFunctionCall(PhpVoidFunctionResultUsedInspection.java:35)
	at com.jetbrains.php.lang.psi.elements.impl.FunctionReferenceImpl.accept(FunctionReferenceImpl.java:48)
	at com.jetbrains.php.lang.psi.elements.impl.PhpASTElementImpl.accept(PhpASTElementImpl.java:26)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:56)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.runToolOnElements(LocalInspectionsPass.java:315)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$3(LocalInspectionsPass.java:281)
	at com.intellij.codeInspection.ex.InspectionEventsKt.reportWhenInspectionFinished(inspectionEvents.kt:9)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$4(LocalInspectionsPass.java:276)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:156)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:148)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$5(LocalInspectionsPass.java:272)
	at com.intellij.util.Ast
```

**Comment by olivernybroe** [comment]
Looks like this is caused by PhpStorm has removed support for `getFQN` on `ClassConstantReferenceImpl`...

@adelf do you have any quick fix idea for a workaround?

**Comment by olivernybroe** [comment]
Scratch that, I think I found a solution 👍

**Comment by adelf** [comment]
Stack trace is not full. I’ll take a look soon. BTW, this is a very early version, people should understand that there are bugs and API changes.

**Comment by adelf** [comment]
Ok )

**Comment by olivernybroe** [comment]
A version has been released and 2021.1 support should be there now.
